src/core/unif_rule.ml

```diff
@@ -16,20 +18,26 @@ open Parsing
     - It is automatically loaded. *)
 let sign : Sign.t =
   let dummy = Sign.dummy () in
-  let sign = {dummy with Sign.sign_path = LpLexer.unif_rule_path} in
-  Sign.loaded := Path.Map.add LpLexer.unif_rule_path sign !(Sign.loaded);
+  let sign = {dummy with Sign.sign_path = path} in
+  Sign.loaded := Path.Map.add path sign !(Sign.loaded);
   sign
 
+(** Parser representation of equiv symbol. *)
+let p_equiv = "≡"
```

Comment:
Why exporting this constant? The name of equiv can be obtained by equiv.sym_name if needed.